sbi/simulators/simutils.py

```diff
@@ -5,14 +5,62 @@
 import torch
 from torch import Tensor
 from tqdm.auto import tqdm
+from sbi.simulators.mp_simulator import simulate_mp
+
+
+def simulate_mp_in_batches(
+    simulator: Callable,
+    theta: Tensor,
+    sim_batch_size: Optional[int],
+    number_of_workers: int = 4,
+    worker_batch_size: int = 20,
+    show_progressbar: Optional[bool] = True,
+):
+    """
+    Distributes all parameters theta to simulate_in_batches, i.e. splits them between
```

Comment:
"Return parameters $\theta$ and corresponding data $x$ simulated using multiprocessing. 

Distributes all parameters theta to simulate_in_batches, i.e. splits them between..."